tests/headers/template_alias_namespace.hpp

```diff
@@ -0,0 +1,13 @@
+// bindgen-flags: --enable-cxx-namespaces -- -std=c++14
```

Comment:
What happens if we don't have `--enable-cxx-namespaces` here? Will the generated rust code be all in the same namespace, or will it fail, or something else?